src/main/java/com/qa/shopkick/appium/ScreenBaseClass.java

```diff
@@ -14,6 +16,38 @@
     public ScreenBaseClass() {
     }
 
+    @AndroidFindBy(uiAutomator = "new UiSelector().resourceId(\"com.shopkick.app:id/button_text\")")
+    protected static WebElement guestIcon;
+
+    @AndroidFindBy(uiAutomator = "new UiSelector().resourceId(\"com.shopkick.app:id/button_image\")")
+    protected static WebElement userIcon;
+    //com.shopkick.app:id/button_image
+
```

Comment:
@mmadhusoodan will we be moving the UI elements to a different page? (Eventually?)